vips/vip-433/bsctestnet.ts

```diff
@@ -0,0 +1,178 @@
+import { BigNumber } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { LzChainId, ProposalType } from "src/types";
+import { makeProposal } from "src/utils";
+
+const { POOL_REGISTRY, VTREASURY, RESILIENT_ORACLE } = NETWORK_ADDRESSES["basesepolia"];
+
+export const COMPTROLLER_CORE = "0x272795dd6c5355CF25765F36043F34014454Eb5b";
+export const wsuperOETHb_Oracle = "0x72050243b23a7f0f74D37e1B85Df9D6486D1a331";
+
+type Token = {
+  address: string;
+  decimals: number;
+  symbol: string;
+};
+
+export const token = {
+  address: "0x02B1136d9E223333E0083aeAB76bC441f230a033",
+  decimals: 18,
+  symbol: "wsuperOETHb",
+};
+
+type Market = {
+  vToken: {
+    address: string;
+    name: string;
+    symbol: string;
+    underlying: Token;
+    decimals: number;
+    exchangeRate: BigNumber;
+    comptroller: string;
+  };
+  riskParameters: {
+    collateralFactor: BigNumber;
+    liquidationThreshold: BigNumber;
+    supplyCap: BigNumber;
+    borrowCap: BigNumber;
+    reserveFactor: BigNumber;
+    protocolSeizeShare: BigNumber;
+  };
+  initialSupply: {
+    amount: BigNumber;
+    vTokenReceiver: string;
+  };
+  interestRateModel: {
+    address: string;
+    base: string;
+    multiplier: string;
+    jump: string;
+    kink: string;
+  };
+};
+
+export const market: Market = {
+  vToken: {
+    address: "0x5aFb0bDdB2cc7d79609AF4F4E1273d14c88c9cBC",
+    name: "Venus wsuperOETHb",
+    symbol: "vwsuperOETHb_Core",
+    underlying: token,
+    decimals: 8,
+    exchangeRate: parseUnits("1", 28),
+    comptroller: COMPTROLLER_CORE,
+  },
+  riskParameters: {
+    collateralFactor: parseUnits("0.73", 18),
+    liquidationThreshold: parseUnits("0.78", 18),
+    supplyCap: parseUnits("2000", 18),
+    borrowCap: parseUnits("0", 18),
+    reserveFactor: parseUnits("0.2", 18),
+    protocolSeizeShare: parseUnits("0.05", 18),
+  },
+  initialSupply: {
+    amount: parseUnits("0.3", 18),
+    vTokenReceiver: VTREASURY,
+  },
+  interestRateModel: {
+    address: "0x43B1cF9AFA734F04d395Db87c593c75c2701Ea49",
+    base: "0",
+    multiplier: "0.09",
+    jump: "3",
+    kink: "0.45",
+  },
+};
+
+export const FIXED_PRICE = parseUnits("1.75254694", 18);
```

Comment:
I think we don't need this, because the underlying token of the of the ERC 4626 oracle (WETH) is already configured in our oracles